tutorials/tour/_posts/2017-02-13-tour-of-scala.md

```diff
@@ -38,10 +38,7 @@ A [local type inference mechanism](local-type-inference.html) takes care that th
 
 ## Scala is extensible ##
 
-In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries:
-
-* any method may be used as an [infix or postfix operator](operators.html)
-* [closures are constructed automatically depending on the expected type](automatic-closures.html) (target typing).
+In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries. Any method may be used as an [infix or postfix operator](operators.html).
 
 A joint use of both features facilitates the definition of new statements without extending the syntax and without using macro-like meta-programming facilities.
```

Comment:
"both" features as in Jasper-M's link to the classic text. Also, by "macro-like", today we mean "macros." Maybe, "without using metaprogramming facilities such as macros." (They don't hyphenate meta-programming on scalameta.org.)